tests/test_parse.py

```diff
@@ -0,0 +1,66 @@
+from lxml import etree 
+from ukrdc_cupid.core.parse.xml_validate import validate_rda_xml_string
+
+def test_transplant_validation():
+    # As the source of big headaches elsewhere putting some tests here
+    v4_00_transplant_xml =  """<ukrdc:PatientRecord xmlns:ukrdc="http://www.rixg.org.uk/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <SendingFacility channelName="UKRDCSampleExtract" schemaVersion="4.0.0" time="2022-07-21T09:00:00">ABC123</SendingFacility>
+    <SendingExtract>UKRDC</SendingExtract>
+        <Patient>
+            <PatientNumbers>
+                <PatientNumber>
+                    <Number>AAA111B</Number>
+                    <Organization>LOCALHOSP</Organization>
+                    <NumberType>MRN</NumberType>
+                </PatientNumber>
+                <PatientNumber>
+                    <Number>1111111111</Number>
+                    <Organization>NHS</Organization>
+                    <NumberType>NI</NumberType>
+                </PatientNumber>
+            </PatientNumbers>
+            <Names>
+                <Name use="L">
+                    <Prefix>Mr</Prefix>
+                    <Family>Surname</Family>
+                    <Given>Forename</Given>
+                </Name>
+            </Names>
+            <BirthTime>2006-05-04T18:13:51.0</BirthTime>
+            <Gender>1</Gender>
+        </Patient>
+        <Procedures>
+            <Transplant>
+                <ProcedureType>
+                    <CodingStandard>SNOMED</CodingStandard>
+                    <Code>19647005</Code>
+                    <Description>PEX</Description>
+                </ProcedureType>
+                <ProcedureTime>2006-05-04T18:13:51.0</ProcedureTime>
+                <EnteredBy>
+                    <CodingStandard>ODS</CodingStandard>
+                    <Code>ABC123</Code>
+                    <Description>Dr Foster</Description>
+                </EnteredBy>
+                <EnteredAt>
+                    <CodingStandard>RR1+</CodingStandard>
+                    <Code>ABC123</Code>
+                    <Description>Test Hospital</Description>
+                </EnteredAt>
+                <UpdatedOn>2006-05-04T18:13:51.0</UpdatedOn>
+                <ExternalId>1222333444</ExternalId>
+                <DateRegistered>2006-01-04</DateRegistered>
+                <DonorType>DBD</DonorType>
```

Comment:
@George-D-S can you see anything wrong with this xml? This test replicates the error I am getting where it seems to expect the the other named items in the schema but not the DonorType.

Reply:
DonorType comes before DateRegistered.

https://github.com/renalreg/resources/blob/master/schema/ukrdc/Transplants/Transplant.xsd

Reply:
Ah yes! Thanks